batch7crud-roll2/src/main/java/com/lftechnology/batch7crud/controller/CommonHttpServlet.java

```diff
@@ -36,4 +36,28 @@ protected int parameterValueAsInt(HttpServletRequest request, int index) {
     String[] paths = getPathParameters(request);
     return Integer.parseInt(paths[index]);
   }
+
+  protected String getPageFromPath(HttpServletRequest request) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "getPageFromPath" is 15 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)